tests/test_constants.py

```diff
@@ -21,39 +37,419 @@ def test_constants(self):
+        # CMD_FIRETV_PROPERTIES_LAZY_NO_RUNNING_APPS
+        self.assertEqual(constants.CMD_FIRETV_PROPERTIES_LAZY_NO_RUNNING_APPS, "(dumpsys power | grep 'Display Power' | grep -q 'state=ON' || dumpsys power | grep -q 'mScreenOn=true') && echo -e '1\c' && dumpsys power | grep mWakefulness | grep -q Awake && echo -e '1\c' && dumpsys power | grep Locks | grep 'size=' && CURRENT_APP=$(dumpsys window windows | grep -E 'mCurrentFocus|mFocusedApp') && CURRENT_APP=${CURRENT_APP#*ActivityRecord{* * } && CURRENT_APP=${CURRENT_APP#*{* * } && CURRENT_APP=${CURRENT_APP%%/*} && CURRENT_APP=${CURRENT_APP%\}*} && echo $CURRENT_APP && (dumpsys media_session | grep -A 100 'Sessions Stack' | grep -A 100 $CURRENT_APP | grep -m 1 'state=PlaybackState {' || echo) && (dumpsys activity starter | grep -E -o '(ExternalTv|HDMI)InputService/HW[0-9]' -m 1 | grep -o 'HW[0-9]' || echo)")
+
+        # CMD_FIRETV_PROPERTIES_NOT_LAZY_RUNNING_APPS
+        self.assertEqual(constants.CMD_FIRETV_PROPERTIES_NOT_LAZY_RUNNING_APPS, "(dumpsys power | grep 'Display Power' | grep -q 'state=ON' || dumpsys power | grep -q 'mScreenOn=true') && echo -e '1\c' || echo -e '0\c' && dumpsys power | grep mWakefulness | grep -q Awake && echo -e '1\c' || echo -e '0\c' && dumpsys power | grep Locks | grep 'size=' && CURRENT_APP=$(dumpsys window windows | grep -E 'mCurrentFocus|mFocusedApp') && CURRENT_APP=${CURRENT_APP#*ActivityRecord{* * } && CURRENT_APP=${CURRENT_APP#*{* * } && CURRENT_APP=${CURRENT_APP%%/*} && CURRENT_APP=${CURRENT_APP%\}*} && echo $CURRENT_APP && (dumpsys media_session | grep -A 100 'Sessions Stack' | grep -A 100 $CURRENT_APP | grep -m 1 'state=PlaybackState {' || echo) && (dumpsys activity starter | grep -E -o '(ExternalTv|HDMI)InputService/HW[0-9]' -m 1 | grep -o 'HW[0-9]' || echo) && ps | grep u0_a")
+
+        # CMD_FIRETV_PROPERTIES_NOT_LAZY_NO_RUNNING_APPS
+        self.assertEqual(constants.CMD_FIRETV_PROPERTIES_NOT_LAZY_NO_RUNNING_APPS, "(dumpsys power | grep 'Display Power' | grep -q 'state=ON' || dumpsys power | grep -q 'mScreenOn=true') && echo -e '1\c' || echo -e '0\c' && dumpsys power | grep mWakefulness | grep -q Awake && echo -e '1\c' || echo -e '0\c' && dumpsys power | grep Locks | grep 'size=' && CURRENT_APP=$(dumpsys window windows | grep -E 'mCurrentFocus|mFocusedApp') && CURRENT_APP=${CURRENT_APP#*ActivityRecord{* * } && CURRENT_APP=${CURRENT_APP#*{* * } && CURRENT_APP=${CURRENT_APP%%/*} && CURRENT_APP=${CURRENT_APP%\}*} && echo $CURRENT_APP && (dumpsys media_session | grep -A 100 'Sessions Stack' | grep -A 100 $CURRENT_APP | grep -m 1 'state=PlaybackState {' || echo) && (dumpsys activity starter | grep -E -o '(ExternalTv|HDMI)InputService/HW[0-9]' -m 1 | grep -o 'HW[0-9]' || echo)")
+
+    def test_current_app_extraction_atv_launcher(self):
```

Comment:
Minor comment, just for the sake of correctness. 

Since the shell commands require `grep` and maybe other Unix commands, this test and the ones below should probably be skipped on Windows. I don't know if they would work on a Mac. 

Not a big deal, though, because they run just fine in GitHub Actions and Travis CI. So it's probably not worth the time to make these changes.

Reply:
It did cross my mind to skip these tests when the operating system would be Windows.
Considering both Python and grep are not native to Windows, it did not seem worth the effort to implement this logic.
Besides nowadays there are so many ways to have GNU utilities available on a Windows machine (e.g. WSL).